pyproject.toml

```diff
@@ -29,23 +107,63 @@ line-length = 88
 line-length = 88
 target-version = "py38"
 fix = true
-src = ["npe2", "tests"]
+src = ["src/npe2", "tests"]
 select = [
-    "E", "F", "W", #flake8
-    "UP", # pyupgrade
-    "I", # isort
-    "B", # flake8-bugbear
-    "C4", # flake8-comprehensions
-    "TID",  # flake8-tidy-imports
-    "RUF",  # ruff-specific rules
+    "E",
+    "F",
+    "W",   #flake8
+    "UP",  # pyupgrade
+    "I",   # isort
+    "B",   # flake8-bugbear
+    "C4",  # flake8-comprehensions
+    "TID", # flake8-tidy-imports
+    "RUF", # ruff-specific rules
 ]
 
 [tool.ruff.per-file-ignores]
-"npe2/cli.py" = ["B008","A00"]
+"src/npe2/cli.py" = ["B008", "A00"]
```

Comment:
Add comments for these?  Something like:

```python
# B008 - (flake8) Do not perform function call {name} in argument defaults
"src/npe2/cli.py" = ["B008", "A00"]
```

I tried to look up A00 but couldn't find it. Just want to double check you didn't mean something like A001.